internal/controller/dragonfly_instance.go

```diff
@@ -198,10 +198,6 @@ func (dfi *DragonflyInstance) configureReplica(ctx context.Context, pod *corev1.
 		return err
 	}
 
-	if err := dfi.updateStatus(ctx, PhaseReady); err != nil {
```

Comment:
We should not update the status here as this might stop the rollout workflow in the middle. if its triggered.